README.md

```diff
@@ -102,6 +102,56 @@ To use the `cics-bundle-maven-plugin`:
 
 1. To include CICS bundleparts like FILE or URIMAP, put the bundlepart files in your Maven project's resources directory, for instance `src/main/resources`. Files in your Maven project's resources directory will be included within the output CICS bundle, and supported types will have a define added to the CICS bundle's cics.xml.
 
+## Install a CICS bundle using `cics-bundle-maven-plugin`
+
+Following the instructions above, you will now have built a CICS bundle. To extend this, you can use the `cics-bundle-maven-plugin` to install this into CICS by using the CICS bundle deployment API. This requires some setup in CICS as a [prerequisite](#Prerequisites).
+
+1. Have your system programmer create your bundle definition in CSD and tell you the CSD group and bundle definition name they have used.
+The bundle directory of your bundle definition should be set as follows: `<bundle_deploy_root>/<bundle_id>_<bundle_version>`. 
+
+1. In the `pom.xml`, extend the plugin configuration to include the extra parameters below:
+
+    ```xml
```

Comment:
The reason the xml code tag wasn't working in the other pages is because it needs to not be indented

Reply:
Strangely this one does work, I've checked the format by looking at it on my branch in a browser. I can remove indents just incase though